apps/condo/domains/common/utils/redis-var.js

```diff
@@ -0,0 +1,29 @@
+const IORedis = require('ioredis')
+const { isString, get } = require('lodash')
+
+const conf = require('@core/config')
+
+const REDIS_URL = conf['REDIS_URL']
+
+class RedisVar {
```

Comment:
really your current task is not required any redis connection. Please don't complicate your task

Reply:
We discussed that I need some place to store `createdAt` of last proceeded receipt, you suggested to use Redis, so I did use it.